web-components/input/text/react.ts

```diff
@@ -0,0 +1,9 @@
+import SvelteToReact, { ReactProps } from "../../svelte-react";
+import TextInput from './text.svelte'
+import type { TextInputEvents, TextInputProps } from './text.svelte'
+
+// Prop types for a svelte component are not able to be inferred at compile-time,
+// only at dev time in typescript plugins, so we must duplicate it here, manually.
+// TODO: have this type definition be auto-generated in the bundler.
```

Comment:
We actually have a PR that does this: https://github.com/brave-experiments/leo/pull/44

I think ideally we should land #44 first and then use it's auto generated React binding.